src/views/Overview.vue

```diff
@@ -0,0 +1,90 @@
+<template>
+  <div class="container">
+    <div class="bar-chart-container">
+      <BarChart />
+    </div>
+    <div class="bar-chart-container2">
+      <BarChartId2/>
+    </div>
+    <div class="bar-chart-container3">
+      <BarChartId3/>
+    </div>
+    <div class="bar-chart-container3">
+      <BarChartId4/>
+    </div>
+    <div class="bar-chart-container3">
+      <BarChartId5/>
+    </div>
+    <div class="bar-chart-container3">
+      <BarChartId6/>
+    </div>
+    <div class="pie-chart-container">
+      <PieChart />
+    </div>
+  </div>
+</template>
+
+<script>
+import BarChart from "@/components/charts/BarChart.vue";
+import PieChart from "@/components/charts/PieChart.vue";
+import BarChartId2 from "@/components/charts/BarChartId2.vue";
+import BarChartId3 from "@/components/charts/BarChartId3.vue";
+import BarChartId4 from "@/components/charts/BarChartId4.vue";
+import BarChartId5 from "@/components/charts/BarChartId5.vue";
+import BarChartId6 from "@/components/charts/BarChartId6.vue";
+
+export default {
+  name: "Overview",
```

Comment:
Adding
```js
    computed: {
        hospitals() {
            return this.$store.getters.allHospitals;
        },
```
Will return an array of all hospitals which can then be passed into the pie chart and bar chart as a prop